web/ui/src/Pools/view-pools.html

```diff
@@ -3,24 +3,24 @@
 <h3 class="pull-left" translate>pools</h3>
 
 <div class="control-buttons pull-right">
-  <span class="add-control">
+    <span class="add-control">
     <button class="btn-link" ng-click="modalAddPool()">
       <span class="glyphicon glyphicon-plus-sign"></span>
-      <span translate>pool</span>
+    <span translate>pool</span>
     </button>
-  </span>
+    </span>
 </div>
 
 <table jelly-table data-data="pools" data-config="poolsTable" class="table">
-    <tr ng-repeat="pool in $data">
-      <td data-title="'pools_tbl_id'|translate" sortable="'id'" ng-click="click_pool(pool.id)" class="link">{{pool.id | cut:true:50}}</td>
-      <td data-title="'core_capacity'|translate" sortable="'model.CoreCapacity'">{{pool.model.CoreCapacity}}</td>
-      <td data-title="'memory_usage'|translate" sortable="'model.MemoryCommitment|toGB'"><span ng-class="{error: pool.model.MemoryCommitment>pool.model.MemoryCapacity}">{{pool.model.MemoryCommitment | toGB}}</span> / {{pool.model.MemoryCapacity | toGB}}</td>
-      <td data-title="'pools_tbl_created_at'|translate" sortable="'model.CreatedAt'">{{pool.model.CreatedAt | date : 'medium'}}</td>
-      <td data-title="'updated_at'|translate" sortable="'model.UpdatedAt'">{{pool.model.UpdatedAt | date : 'medium'}}</td>
-      <td data-title="'pools_tbl_actions'|translate"><button ng-click="clickRemovePool(pool.id)" ng-class="{disabled: isDefaultPool(pool.id)}" class="btn btn-link action" name="{{pool.id}}">
+  <tr ng-repeat="pool in $data">
+    <td data-title="'pools_tbl_id'|translate" sortable="'id'" ng-click="click_pool(pool.id)" class="link">{{pool.id | cut:true:50}}</td>
+    <td data-title="'core_capacity'|translate" sortable="'model.CoreCapacity'">{{pool.model.CoreCapacity}}</td>
+    <td data-title="'memory_usage'|translate" sortable="'model.MemoryCapacity'"><span ng-class="{error: pool.model.MemoryCommitment>pool.model.MemoryCapacity}">{{pool.model.MemoryCommitment | toGB}}</span> / {{pool.model.MemoryCapacity | toGB}}</td>
```

Comment:
any particular reason to change sorting to capacity instead of commitment? removing the `toGB` filter is a good idea though.